dspace-api/src/main/java/org/dspace/authenticate/ShibAuthentication.java

```diff
@@ -332,6 +328,8 @@ public int authenticate(Context context, String username, String password,
 							affiliation = affiliation.substring(index+1, affiliation.length());
 						}
 					} 
+                                        // Replace spaces with underscores
+					affiliation = affiliation.replace(" ", "_");
```

Comment:
Thank you for this hint!
I can confirm that its working with escaped spaces, so this change is really not necessary at all. But the documentation might get improved.